scripts/ci/ie_setup.bat

```diff
@@ -45,3 +45,13 @@ reg add "HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\Main" ^
 :: Disable cross-origin XHRs. Sauce enables this in IE by default.
 @ECHO OFF
 REG ADD "HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Zones\3" /v 1406 /t REG_DWORD /d 3 /f
+
+@echo off
+setlocal
+
+set "URL=https://raw.githubusercontent.com/shicks/closure-library-temp/master/CloseAdobeDialog.exe"
```

Comment:
It doesn't. @shicks and I were working together on this and he just hosted it in repo for convenience. I definitely won't commit it like this. Once we verify the script works, we'll point this to master.